packages/react/src/primitives/SelectField/SelectField.tsx

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { ComponentClassNames, useAmplifyFieldID } from '../shared';
+import { FieldErrorMessage } from '../Field';
+import { Flex } from '../Flex';
+import { Select } from '../Select';
+import { Label } from '../Label';
+import { SelectFieldProps } from '../types/selectField';
+
+export const SelectField: React.FC<SelectFieldProps> = (props) => {
+  const {
+    alignContent,
+    alignItems,
+    children,
+    className,
+    defaultValue,
+    direction = 'column',
+    errorMessage,
+    gap,
+    hasError = false,
+    icon,
+    iconColor,
+    id,
+    isDisabled,
+    isRequired,
+    justifyContent,
+    label,
+    labelHidden = false,
+    onChange,
+    placeholder,
+    size,
+    testId,
+    value,
+    variation,
+    wrap,
+    ...rest
+  } = props;
+
+  const fieldId = useAmplifyFieldID(id);
+
+  return (
+    <Flex
+      alignContent={alignContent}
+      alignItems={alignItems}
+      className={classNames(ComponentClassNames.Field, className)}
+      data-size={size}
+      direction={direction}
+      gap={gap}
+      justifyContent={justifyContent}
+      testId={testId}
+      wrap={wrap}
+    >
+      <Label htmlFor={fieldId} visuallyHidden={labelHidden}>
+        {label}
+      </Label>
+      <Select
```

Comment:
IMO, It looks a bit wired that there is a `FieldDescription` above a `Select` because there will always be a placeholder option there. But for consistency, I am good to include that